crates/rolldown/src/module_loader/module_task.rs

```diff
@@ -146,7 +155,7 @@ impl ModuleTask {
         is_user_defined_entry: self.is_user_defined_entry,
         replace_global_define_config: self.ctx.meta.replace_global_define_config.clone(),
       },
-      CreateModuleViewArgs { source: source.clone(), sourcemap_chain, hook_side_effects },
+      CreateModuleViewArgs { source, sourcemap_chain, hook_side_effects },
```

Comment:
After this pr, the source here is always empty if `moduleType` is CSS?

Reply:
Actually not. `EcmaModuleFactory won't use this passed `source`, instead, it just use `export {}` as the input to be parsed and don't care about passed `source`.